maintain.js

```diff
@@ -0,0 +1,129 @@
+import fetch from "node-fetch"
+import chalk from "chalk"
+import puppeteer from "puppeteer"
+
+import { readFileSync, writeFileSync } from "fs"
+
+const members = JSON.parse(readFileSync("members.json", { encoding: "utf8" }))
+
+let activeMembers = []
+
+const webringLink = "https://webring.hackclub.com"
+
+async function checkWithBrowser(url, webringLink) {
```

Comment:
nitpick - would be nice to add a comment to this function describing what it actually does